spec/factories/locations.rb

```diff
@@ -1,7 +1,16 @@
 FactoryBot.define do
   factory :location do
-    name { "MyString" }
+    sequence(:name) { |n| "Location #{n}" }
     facility
     organization
+
+    trait :with_enclosures do
+      transient { enclosures_count { 5 } }
```

Comment:
I think `1` might be better defualt here. I worry that if we have to use this trait alot we'd end up creating alot of uneeded records, slowing down the test suit as a whole.

Reply:
Makes sense